src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11006,27 +11004,27 @@ void func_80841A50(PlayState* play, Player* this) {
     }
 }
 
-typedef void (*PlayerInitModeFunc)(PlayState*, Player*);
+typedef void (*PlayerStartModeFunc)(PlayState*, Player*);
 
 // Initialisation functions for various gameplay modes depending on spawn params.
 // There may be at most 0x10 due to it using a single nybble.
-PlayerInitModeFunc sPlayerInitModeFuncs[PLAYER_INITMODE_MAX] = {
-    Player_InitMode_0,         // PLAYER_INITMODE_0
-    Player_InitMode_1,         // PLAYER_INITMODE_1
-    Player_InitMode_2,         // PLAYER_INITMODE_2
-    Player_InitMode_3,         // PLAYER_INITMODE_3
-    Player_InitMode_4,         // PLAYER_INITMODE_4
-    Player_InitMode_5,         // PLAYER_INITMODE_5
-    Player_InitMode_6,         // PLAYER_INITMODE_6
-    Player_InitMode_7,         // PLAYER_INITMODE_7
-    func_80841744,             // PLAYER_INITMODE_8
-    func_80841744,             // PLAYER_INITMODE_9
-    func_8083ADF0,             // PLAYER_INITMODE_A
-    Player_InitMode_B,         // PLAYER_INITMODE_B
-    Player_InitMode_Telescope, // PLAYER_INITMODE_TELESCOPE
-    Player_InitMode_D,         // PLAYER_INITMODE_D
-    func_8083ADF0,             // PLAYER_INITMODE_E
-    Player_InitMode_F,         // PLAYER_INITMODE_F
+PlayerStartModeFunc sStartModeFuncs[PLAYER_START_MODE_MAX] = {
+    Player_StartMode_0,         // PLAYER_START_MODE_0
+    Player_StartMode_1,         // PLAYER_START_MODE_1
+    Player_StartMode_2,         // PLAYER_START_MODE_2
+    Player_StartMode_3,         // PLAYER_START_MODE_3
+    Player_StartMode_4,         // PLAYER_START_MODE_4
+    Player_StartMode_5,         // PLAYER_START_MODE_5
+    Player_StartMode_6,         // PLAYER_START_MODE_6
+    Player_StartMode_7,         // PLAYER_START_MODE_7
+    Player_StartMode_WarpTag,   // PLAYER_START_MODE_8
+    Player_StartMode_WarpTag,   // PLAYER_START_MODE_9
+    Player_StartMode_E,         // PLAYER_START_MODE_A
```

Comment:
Does not matter what so ever, but any particular reason to name the function after E instead of A?

Reply:
It's because A is unused while E is used. See here:
https://github.com/zeldaret/oot/pull/2301#discussion_r1846355649

It's identical to what OoT just named `Player_StartMode_MoveForwardSlow, // PLAYER_START_MODE_MOVE_FORWARD_SLOW`